### PR TITLE
bluetooth: switch to pydbus

### PR DESCRIPTION
**Pull request by maximbaz**
I got an impression that we are slowly switching to `pydbus`, this is just one tiny step towards getting rid of `dbus` dependency.

### CODE REVIEW

File: py3status/modules/bluetooth.py [hunk 1]
```diff
@@ -121,7 +118,7 @@ def post_config_hook(self):
     def bluetooth(self):
         try:
             devices = get_connected_devices()
-        except dbus.DBusException:
+        except Exception:
```
Comment (lasers) [inline py3status/modules/bluetooth.py]: I can't trigger this. Switching to `pydbus` might make `try/except` + `STRING_NOT_STARTED` useless cruft.

File: py3status/modules/bluetooth.py [hunk 2]
```diff
@@ -29,18 +29,15 @@
 {'color': '#FF0000', 'full_text': u'BT'}
 """
 
-import dbus
+from pydbus import SystemBus
```
Comment (lasers) [inline py3status/modules/bluetooth.py]: This module ought to have dbus `Requires:` too just like kdeconnector, mpris, systemd, vpn_status, wwan.

File: py3status/modules/bluetooth.py (hunk 1 above)
Comment (maximbaz) [reply py3status/modules/bluetooth.py]: Same, I'll just get rid of this entirely then

Comment (lasers) [reply py3status/modules/bluetooth.py]: Bump.

**Comment by ultrabug** [comment]
Gentle bump @maximbaz plz?

**Comment by maximbaz** [comment]
Sorry for the silence guys, been really busy and couldn't check this PR, big thanks for taking care of it!

**Pull request merged**